js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -65,100 +65,43 @@ import {EmptyDAGNotice, EntirelyFilteredDAGNotice, LoadingNotice} from '../pipel
 import {ExplorerPath} from '../pipelines/PipelinePathUtils';
 import {GraphQueryInput} from '../ui/GraphQueryInput';
 import {Loading} from '../ui/Loading';
-import {WorkspaceContext} from '../workspace/WorkspaceContext';
 
 type AssetNode = AssetNodeForGraphQueryFragment;
 
-type Filters = {
-  groups?: AssetGroupSelector[];
-  computeKindTags?: string[];
-  changedInBranch?: ChangeReason[];
-  tags?: DefinitionTag[];
-  owners?: AssetOwner[];
-};
-type OptionalFilters =
-  | {
-      filters: Filters;
-      setFilters: React.Dispatch<React.SetStateAction<Filters>>;
-    }
-  | {filters?: null; setFilters?: null};
-
 type Props = {
   options: GraphExplorerOptions;
   setOptions?: (options: GraphExplorerOptions) => void;
 
+  assetFilterState?: AssetFilterState;
```

Comment:
I know it's a stupid nit but I didn't realize at first that this was more than the `type FIlters` above, and actually contains both the filters and their setters.  I wonder if just calling it `assetFilters` (leaving off "state") would make it more clear that it's a full interface?

Also I really like how this cleans up this page, it's so much more readable with all the state stuff moved out.

Reply:
The thing is I started refactoring the actual hooks too and realized that I would actually want to call that abstraction AssetFilters, while this one is just the state and updating the state.